### DISCUSSION TITLE
Progress Report #1

### DISCUSSION THREAD

**Discussion by ImproveAbility** [ATMakersOrg/EyeAlert#18] [Ideas]
See link for video.

Update on current four ideas:
1. IR watchdog - working prototype - Ron will send one to Antoinette and one to Nachum for testing.  Testing to focus on how device has to be positioned to pick up IR from camera.
2. Stand alone device - once Raspberry Pi 4 obtained will be able to move forward.
3. Project Activate - see update in other discussion - https://github.com/ATMakersOrg/EyeAlert/discussions/14
4. Sensitive switches - Ron needs help from Bill - no current progress on this yet.

AV sent link to Bill to post on socials for additional comment.

**Comment by nelsonii** [comment]
There are two folks we should reach out to re: sensitive switches. 

Dr. Jeff Ebin out of Houston area has had success with his versions of facial movement solutions -- he's working with patients with very limited movement. He's part of the ATMakers and Assistive Technology groups on Facebook.

Graham Law, with Celtic Magic, in the UK, has also done some very sensitive switching devices. He has an accelerometer based solution (finger or head motion) and has a ITouch device which using IR to pick up very small facial movements. His solutions are very well thought out -- multiple mounting options, calibration, etc. He's done a lot of work with the math side of things -- determining the proper deltas to tell the difference between a desired command and general movement. He's not on Facebook much, but Jeff and I have both been in contact with him.

**Reply by nelsonii** [reply]
https://www.youtube.com/watch?v=9rbxdSq8i3Y
